portfolio-project-matching-app/components/Button.js

```diff
@@ -1,12 +1,12 @@
 import styles from './Button.module.css';
 
-const Button = ({text, type='btn', onClick}) => {
+const Button = ({text, type='btnGeneral', onClick, addClassName=""}) => {
```

Comment:
Super smart idea. No matter how much we try to stick to the cookie cutter component, it  always helps to be able to inject some context-specific styling. Love it.